djangocms_versioning/monkeypatch/extensions.py

```diff
@@ -0,0 +1,26 @@
+from cms.extensions.extension_pool import ExtensionPool
+from cms.models.titlemodels import PageContent
+
+
+def _copy_title_extensions(self, source_page, target_page, language, clone=False):
+    # the line below has been modified to accomodate versioning.
```

Comment:
Can we add a comment as either a docstring or above the function explaining why we have made changes here.

Something like:  The existing method ExtensionPool._copy_title_extensions will only ever get published versions, we change the queries to get the latest draft version.